presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -510,6 +510,17 @@ private static LongWriteFunction timestampWriteFunction(TimestampType timestampT
         };
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        String sql = format(
+                "COMMENT ON COLUMN %s.%s IS '%s'",
```

Comment:
it'd be better to generate explicit `COMMENT ... IS NULL` when `comment.isEmpty()`